files/image_config/kubernetes/kubernetes.list

```diff
@@ -0,0 +1 @@
+deb https://apt.kubernetes.io/ kubernetes-xenial main
```

Comment:
why xenial, this is an ubuntu release?

Reply:
I will check it out. I am not pretty familiar in this area.

Reply:
Ref: [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#k8s-install-0)

For Ubuntu, Debian & HypriotOS:

sudo apt-get update && sudo apt-get install -y apt-transport-https curl
curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
cat <<EOF | sudo tee /etc/apt/sources.list.d/kubernetes.list
deb https://apt.kubernetes.io/ kubernetes-xenial main
EOF
sudo apt-get update
sudo apt-get install -y kubelet kubeadm kubectl
sudo apt-mark hold kubelet kubeadm kubectl

Reply:
It appears to have [debian-installer](https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/debian-installer).
Ref: [here](https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main)

Reply:
If we must use `xenial`, add code comment to about Debian Stretch, so we will find it when we upgrade OS.

---
In reply to: [403753349](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r403753349) [](ancestors = 403753349)

Reply:
Good point. Added comments. Also we would be adding test cases, that would double confirm that it matches current Debian version.